support/Procfile

```diff
@@ -1,9 +1,8 @@
-web: support/builder_web.sh
-postgres: support/db/start.sh
-api: target/debug/bldr-api start --path /tmp/depot
-admin: target/debug/bldr-admin start
-router: target/debug/bldr-router start
-jobsrv: target/debug/bldr-jobsrv start
-sessionsrv: target/debug/bldr-sessionsrv start
-originsrv: target/debug/bldr-originsrv start
-worker: target/debug/bldr-worker start
+postgres: hab svc start core/builder-datastore
+api: /src/target/debug/bldr-api start --path /tmp/depot --config /hab/svc/builder-api/user.toml
```

Comment:
The '/tmp/depot' folder can sometimes get wiped out - we might want to consider changing it to some other location - we could even remove the --path param here, and instead specify in it the user.toml.

Also, the config file locations here should match what is in the BUILDER_DEV.md doc, so that should be updated also,